ci-operator/config/openshift-psap/special-resource-operator/openshift-psap-special-resource-operator-master.yaml

```diff
@@ -27,8 +27,13 @@ images:
   from: src
   to: test-bin
 promotion:
-  excluded_images:
-  - test-bin
+<<<<<<< HEAD
+=======
```

Comment:
You need to resolve the rebase conflict before you push, please choose the code you want are remove those lines.